test/test-fs-copyfile.c

```diff
@@ -188,7 +188,7 @@ TEST_IMPL(fs_copyfile) {
   unlink(dst);
   r = uv_fs_copyfile(NULL, &req, fixture, dst, UV_FS_COPYFILE_FICLONE_FORCE,
                      NULL);
-  ASSERT(r == 0 || r == UV_ENOSYS || r == UV_ENOTSUP);
+  ASSERT(r == 0 || r == UV_ENOSYS || r < 0);
```

Comment:
With the current implementation, I think this could be reduced to `ASSERT(r <= 0);`, but I'm not sure how helpful that is. If we return `UV_ENOTSUP`, this doesn't need to change at all (I think).

Reply:
Thanks for the pointer, I updated the PR with your suggestion.